4337/contracts/test/WebAuthnSigner.sol

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+/* solhint-disable one-contract-per-file */
+pragma solidity >=0.8.0;
+
+import {FCL_WebAuthn} from "./FCL/FCL_Webauthn.sol";
+import {IUniqueSignerFactory} from "./SafeSignerLaunchpad.sol";
+
+struct SignatureData {
+    bytes authenticatorData;
+    bytes clientData;
+    uint256 challengeOffset;
+    uint256[2] rs;
+}
+
+function checkSignature(bytes memory data, bytes calldata signature, uint256 x, uint256 y) view returns (bytes4 magicValue) {
```

Comment:
TIL one can define functions outside the contract

Reply:
Since 0.8